bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/Teaser.java

```diff
@@ -36,71 +36,93 @@
+     * Name of the child node where the Call-to-Action elements are stored
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String NN_CTAS = "ctas";
+    String NN_ACTIONS = "actions";
 
     /**
-     * Name of the property that will store if the image link should be hidden.
+     * Name of the resource property that stores the Call-to-Action link
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_HIDE_IMAGE_LINK = "hideImageLink";
+    String PN_ACTION_LINK = "link";
 
     /**
-     * Name of the property that will store if the title should be hidden.
+     * Name of the resource property that stores the Call-to-Action text
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_HIDE_TITLE = "hideTitle";
+    String PN_ACTION_TEXT = "text";
 
     /**
-     * Name of the property that will store if the title link should be hidden.
+     * Name of the policy property that stores if the Call-to-Action is disabled
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_HIDE_TITLE_LINK = "hideTitleLink";
+    String PN_ACTIONS_DISABLED = "disabled";
 
     /**
-     * Name of the property that will store if the title value should be taken from the linked page.
+     * Name of the policy property that will store if the image link should be hidden.
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_TITLE_VALUE_FROM_PAGE = "titleValueFromPage";
+    String PN_IMAGE_LINK_HIDDEN = "imageLinkHidden";
 
     /**
-     * Name of the property that will store if the description should be hidden.
+     * Name of the policy property that will store if the title should be hidden.
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_HIDE_DESCRIPTION = "hideDescription";
+    String PN_TITLE_HIDDEN = "titleHidden";
 
     /**
-     * Name of the property that will store if the description link should be hidden.
+     * Name of the policy property that will store if the title link should be hidden.
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_HIDE_DESCRIPTION_LINK = "hideDescriptionLink";
+    String PN_TITLE_LINK_HIDDEN = "titleLinkHidden";
 
     /**
-     * Name of the property that will store if the description value should be taken from the linked page.
+     * Name of the resource property that will store if the title value should be taken from the linked page.
      *
      * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
      */
-    String PN_DESCRIPTION_VALUE_FROM_PAGE = "descriptionValueFromPage";
+    String PN_TITLE_FROM_PAGE = "titleFromPage";
+
+    /**
+     * Name of the policy property that will store if the description should be hidden.
```

Comment:
I'd rename:
Name of the policy property that stores if the description is hidden.

Reply:
@jckautzmann & @richardhand, please check https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/179/commits/172afc21185965b6cd070f4cb991cc59ceb2fd4b